.gitignore

```diff
@@ -77,13 +77,6 @@ dist
 crash.log
 crash.*.log
 
-# Exclude all .tfvars files, which are likely to contain sensitive data, such as
```

Comment:
I have no idea why this was in here in the first place. We don't have any local tfvars as of yet

Reply:
I think it's from a default, and it's not unreasonable, it's _very_ easy to accidentally commit tfvars files, and they can be very sensitive. 

I'd rather here explicitly include the 4 files we do want rather than exclude from gitignore all tfvars files

Reply:
I've never used tfvars for sensitive values etc, but sure I will change to that :)